reana_client/api/client.py

```diff
@@ -357,7 +357,7 @@ def upload_to_server(self, workflow, paths, access_token):
 
             # Check if input is an absolute path and upload file.
             else:
-                with open(path) as f:
+                with open(path, 'rb') as f:
```

Comment:
Can you check all the other `open()` places in the client code base?  E.g. CWL runner...